readme.md

```diff
@@ -110,6 +110,16 @@
 
 ## Strategy
 
+### [Backgammon](https://en.wikipedia.org/wiki/Backgammon)
+
+> It is one of the oldest known board games. It is a two player game where each player has fifteen pieces (checkers) which move between twenty-four triangles (points) according to the roll of two dice. The objective of the game is to be first to bear off, i.e. move all fifteen checkers off the board.
+
+![Backgammon](https://upload.wikimedia.org/wikipedia/commons/thumb/3/30/Backgammon_lg.png/440px-Backgammon_lg.png)
```

Comment:
To keep more consistent with the other image sizes, we can use a different prepared size of the same image; one upside to finding an image from wikimedia.

```suggestion
![Backgammon](https://upload.wikimedia.org/wikipedia/commons/thumb/3/30/Backgammon_lg.png/320px-Backgammon_lg.png)
```